media-libs/libsmf/libsmf-1.3.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="A C library for handling SMF files"
+DESCRIPTION="LibSMF is a BSD-licensed C library for handling \
+SMF (*.mid) files. It transparently handles conversions between time \
+and pulses, tempo map handling etc. The only dependencies are C \
+compiler and glib. Full API documentation and examples are included"
+HOMEPAGE="https://sourceforge.net/projects/libsmf/"
+COPYRIGHT="2007-2008 Edward Tomasz Napieraa \
```

Comment:
The back slash is back, at the end of the first line of COPYRIGHT.
(Please remove it, with the white space that precedes it.)
All the rest is perfect!